__tests__/app.test.js

```diff
@@ -36,10 +49,40 @@ afterAll(() => {
         return request(app)
         .get("/api/reviews")
         .expect(200)
-        .then(({})=>{
-
-
+        .then(({body})=>{
+            const { reviews } = body;
+            expect(reviews.length).toBe(13)
+            expect(reviews).toBeInstanceOf(Array)
+            expect(reviews[0]).toBeInstanceOf(Object);
+            reviews.forEach((review)=>{
+                expect(review).toEqual(
+                    expect.objectContaining({
+                        review_id: expect.any(Number),
+                        title: expect.any(String),
+    category: expect.any(String),
+    designer: expect.any(String),
+    owner: expect.any(String),
+    review_body: expect.any(String),
```

Comment:
The ticket states that we shouldn't be getting back a 'review_body', so can we make sure this isn't a property on the review object please?